lib/_tls_wrap.js

```diff
@@ -271,6 +271,14 @@ var proxiedMethods = [
   'setPendingInstances'
 ];
 
+// Proxy HandleWrap, PipeWrap and TCPWrap methods
+proxiedMethods.forEach(function(name) {
+  tls_wrap.TLSWrap.prototype[name] = function methodProxy() {
+    if (this._parent[name])
+      return this._parent[name].apply(this._parent, arguments);
+  };
+});
```

Comment:
do we care about the extra performance lost to creating a closure and referring to the call as `this._parent[name]` instead of `this._parent.<name>`?